src/pages/HomePage/HomePage.tsx

```diff
@@ -1,30 +1,20 @@
 import React from "react";
-import {
-  IonContent,
-  IonHeader,
-  IonPage,
-  IonTitle,
-  IonToolbar,
-} from "@ionic/react";
+import { IonContent, IonHeader, IonPage, IonTitle } from "@ionic/react";
 import "./HomePage.css";
 import StudyList from "../../components/StudyList/StudyList";
+import ThemedToolbar from "../../components/ThemedToolbar/ThemedToolbar";
 
 const HEADER_TEXT = "NMDC Field Notes";
```

Comment:
I think we've used the phrase "NMDC Field Notes" in some places and "Field Notes" in others. I'll create an issue about reconciling all of them. I do not want to hold up this merge for this (and it may end up that it is a non-issue).

Reply:
I created an issue about this, here: #77